eng/tools/oav-runner/src/cli.ts

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+
+import { checkSpecs, checkExamples } from "./runner.js";
+import {
+  outputAnnotatedErrors,
+  outputErrorSummary,
+  outputSuccessSummary,
+  ReportableOavError,
+} from "./formatting.js";
+import { consoleLogger } from "@azure-tools/specs-shared/logger";
```

Comment:
```suggestion
```

I'm not sure this adds any value over the builtin `console` methods like `console.log`.  `specs-shared/logger` is mainly useful when you need to pass an `ILogger` instance to a shared component like `SpecModel`.  Or if you know you are running in a GitHub Action, and want to use their logger.

https://github.com/Azure/azure-rest-api-specs/blob/0752d8ee3182a9b35ae1fa6a4b94962b7642b087/.github/workflows/src/core-logger.js

Reply:
Ok. It's crashing in the latest merge from main anyway.